tsconfig.json

```diff
@@ -13,7 +13,7 @@
     "moduleResolution": "node",
     "noEmit": true,
     "noFallthroughCasesInSwitch": true,
-    "noImplicitReturns": true,
+    "noImplicitReturns": false,
```

Comment:
Why was this changed?

Reply:
In `./IconSet.store'` file there are some functions in which there are no returns for certain control flows. These functions are consumed by the reducer so I can't put some dummy value for every flow, hence I changed this rule.